packages/atlas/src/providers/nftActions/provider.tsx

```diff
@@ -36,17 +36,26 @@ export const NftActionsProvider: React.FC = ({ children }) => {
       decrementOverlaysOpenCount()
     }
   }, [cachedCurrentAction, currentAction, decrementOverlaysOpenCount, incrementOverlaysOpenCount])
-  // --END--
```

Comment:
Same as above - I would keep these comments.

Reply:
Actually, @kdembler  asked me to remove this https://github.com/Joystream/atlas/pull/2307#discussion_r832169316

Reply:
Ahh my bad, I thought the PR where this was removed was merged already! let's keep it then!